include/Core/Models/SpentOutput.h

```diff
@@ -92,8 +92,7 @@ class SpentOutputs : public Traits::ISerializable
 
 	static SpentOutputs Deserialize(ByteBuffer& byteBuffer)
 	{
-		const uint8_t version = byteBuffer.ReadU8();
-		assert(version == 0);
+		assert(byteBuffer.ReadU8() == 0);
```

Comment:
Revert this change. If compiled with `NDEBUG` (many release builds), anything inside of assertions isn't executed, so it will deserialize incorrectly.